deployer/src/handlers/mod.rs

```diff
@@ -55,6 +55,11 @@ pub struct PaginationDetails {
     pub limit: Option<u32>,
 }
 
+#[derive(Deserialize)]
+struct LogSize {
+    head: Option<u32>,
+    tail: Option<u32>,
```

Comment:
I think we can use an enum for the mode here as well, see https://stackoverflow.com/a/74965172